### PR TITLE
Pcapminer

### PR DESCRIPTION
**Pull request by trorabaugh**
Cleaned up the code a bit and changed the entry context to not include specific fields as to not fill up the context.

### CODE REVIEW

File: Scripts/script-PCAPMiner.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  filePath = demisto.executeCommand("getFilePath", {"id": demisto.args()["entryId"] })
+  readyPath = filePath[0]["Contents"]["path"]
+  ret = []
+  filename = filePath[0]["Contents"]["name"]
+  rPath = "/var/lib/demisto/" + readyPath
+  miner = pcap_miner(rPath)
+  jsonResults = miner.summary2json()
+  pyResults = json.loads(jsonResults)
+
+  def saveFileResult(filename, data):
```
Comment (Spivaka) [inline Scripts/script-PCAPMiner.yml]: functions should follow imports

**Comment by Spivaka** [comment]
@trorabaugh ?

**Comment by trorabaugh** [comment]
@Spivaka - I will update either Friday or this weekend ok?

File: Scripts/script-PCAPMiner.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
+commonfields:
+  id: f8b9bff1-aaa5-4fb4-8306-10d97828395c
```
Comment (dorsha) [inline Scripts/script-PCAPMiner.yml]: id should be like the name (and not uuid)

File: Scripts/script-PCAPMiner.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+tags:
+- enhancement
+enabled: true
+args:
+- name: entryId
+  required: true
+  default: true
+  description: Enter the EntryID of the file
+scripttarget: 0
+timeout: 360ns
```
Comment (dorsha) [inline Scripts/script-PCAPMiner.yml]: I think timeout of 360ns is too short...

File: Scripts/script-PCAPMiner.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
+commonfields:
+  id: f8b9bff1-aaa5-4fb4-8306-10d97828395c
+  version: -1
+name: PCAPMiner
```
Comment (dorsha) [inline Scripts/script-PCAPMiner.yml]: script description is missing

File: Scripts/script-PCAPMiner.yml (hunk 1 above)
Comment (trorabaugh) [inline Scripts/script-PCAPMiner.yml]: Will Fix

File: Scripts/script-PCAPMiner.yml (hunk 2 above)
Comment (trorabaugh) [inline Scripts/script-PCAPMiner.yml]: Thx didn't know that

File: Scripts/script-PCAPMiner.yml (hunk 3 above)
Comment (trorabaugh) [inline Scripts/script-PCAPMiner.yml]: Great Point!

File: Scripts/script-PCAPMiner.yml (hunk 4 above)
Comment (trorabaugh) [inline Scripts/script-PCAPMiner.yml]: Yeap slapped together!

**Comment by Spivaka** [comment]
Reopen when you have time to address comments.

**Pull request merged**